app/models/user.rb

```diff
@@ -1,7 +1,6 @@
 # An event participant
 class User < ApplicationRecord
-  validates :name, presence: true
-  validates :email, presence: true
+  validates_presence_of :name, :email
```

Comment:
`validates_presence_of` is the older version, no?

Reply:
You're right, but when I made my first pull request, this was changed to the `validates` instead of `validates_presence_of`.

Reply:
You're right, but when I made my first pull request, this was changed to the `validates` instead of `validates_presence_of`.